packages/cli/README.md

```diff
@@ -334,9 +347,175 @@ ARGUMENTS
 
 OPTIONS
   -c, --ceramic=ceramic  Ceramic API URL
-  -k, --key=key    DID Key
+  -k, --key=key          DID Key
+```
+
+### `glaze pin:add STREAMID`
+
+pin a stream
+
+```
+USAGE
+  $ glaze pin:add STREAMID
+
+ARGUMENTS
+  STREAMID  ID of stream to be pinned
+
+OPTIONS
+  -c, --ceramic=ceramic  Ceramic API URL
+  -k, --key=key          DID Key
+```
+
+### `glaze pin:ls [STREAMID]`
+
+list pinned streams
+
+```
+USAGE
+  $ glaze pin:ls [STREAMID]
+
+ARGUMENTS
+  STREAMID  optional stream ID filter
+
+OPTIONS
+  -c, --ceramic=ceramic  Ceramic API URL
+  -k, --key=key          DID Key
+```
+
+### `glaze pin:rm STREAMID`
+
+unpin a stream
+
+```
+USAGE
+  $ glaze pin:rm STREAMID
+
+ARGUMENTS
+  STREAMID  ID of stream to be unpinned
+
+OPTIONS
+  -c, --ceramic=ceramic  Ceramic API URL
+  -k, --key=key          DID Key
+```
+
+### `glaze stream:commits STREAMID`
+
+list commits contained within a stream
+
+```
+USAGE
+  $ glaze stream:commits STREAMID
+
+ARGUMENTS
+  STREAMID  ID of the stream
+
+OPTIONS
+  -c, --ceramic=ceramic  Ceramic API URL
+  -k, --key=key          DID Key
+```
+
+### `glaze stream:state STREAMID`
+
+get the state of a Stream
+
+```
+USAGE
+  $ glaze stream:state STREAMID
+
+ARGUMENTS
+  STREAMID  ID of the Stream
+
+OPTIONS
+  -c, --ceramic=ceramic  Ceramic API URL
+  -k, --key=key          DID Key
+```
+
+### `glaze tile:create CONTENT`
+
+create a new Tile stream
+
+```
+USAGE
+  $ glaze tile:create CONTENT
+
+ARGUMENTS
+  CONTENT  stream contents (JSON encoded as string)
+
+OPTIONS
+  -c, --ceramic=ceramic    Ceramic API URL
+  -g, --only-genesis       only generate genesis commit
+  -k, --key=key            DID Key
+  -m, --metadata=metadata  stream metadata
+```
+
+### `glaze tile:deterministic METADATA`
+
+load a deterministic Tile stream
+
+```
+USAGE
+  $ glaze tile:deterministic METADATA
+
+ARGUMENTS
+  METADATA  stream metadata
+
+OPTIONS
+  -c, --ceramic=ceramic  Ceramic API URL
+  -k, --key=key          DID Key
```

Comment:
hmm actually I'm wondering now, even for operations that do writes to streams, how would you use `--key`?  What do you provide?  If you just provide the did string, then you don't actually have access to the underlying private key?  Can you actually use this to control what key is used to sign the messages?

Reply:
The value provided should be the private key, not the public one.

Reply:
oh interesting.  Would be good to make that clear in the docs

Reply:
Agreed, I'll make a note of this in the updated documentation.